docs/report/report.tex

```diff
@@ -793,10 +794,20 @@ \section{Technische Umsetzung}
   Zu Beginn des Projekts diskutierte das Team die Technologien, mit denen die \textit{leek-box} umgesetzt werden sollte.
   Der Kunde ließ dem Projektteam hierbei sehr viele Freiheiten.
 
+  \subsection{Hardware}
+  \subsubsection*{Raspberry Pi}
 
+  \subsubsection{NFC Tags \& NFC Reader}
+    Ähnlich wie bei Konkurrenzprodukten, wird als Peripheriegerät für den Raspberry Pi ein NFC Reader genutzt.
+    Die Steuerung der Box hat mittels NFC-Tags durchgeführt werden.
```

Comment:
```suggestion
    Die Steuerung der Box hat mittels NFC-Tags durchgeführt zu werden.
```
Klingt aber auch so noch komisch

Reply:
```suggestion
    Die Steuerung der Box kann so mittels NFC-Tags durchgeführt werden.
```

Reply:
how about that?

Reply:
Find ich gut. Besser als "Hat" das klingt so bestimmend ^^